src/main/webapp/js/common.js

```diff
@@ -255,14 +263,20 @@ function sortTable(oneOfTableCell, colIdx, comp, ascending, row) {
         // $.trim trims leading/trailing whitespaces
         // jQuery(...).text() works like .innerText, but works in Firefox (.innerText does not)
         // $RowList[i].cells[colIdx - 1] is where we get the table cell from
-        var innerText = $.trim(jQuery($RowList[i].cells[colIdx - 1]).text());
+        // If shouldConsiderToolTip is true, we consider the tooltip value instead of innerText
+        if (shouldConsiderToolTip) {
```

Comment:
We can also collapse this into

```
textToCompare = shouldConsiderToolTip ? x : y
```

Also, we can use back ```$``` instead of ```jQuery```, they're the same object